js/page.js

```diff
@@ -4,10 +4,30 @@ $('#small-nav-dropdown').change(function() {
     .val()
 })
 
+const site_tag = 'UA-62780441-30';
+function loadAnalytics(gtag) {
+  // set cookie to expire in 12 x 28 days
+  gtag('config', site_tag, { 'anonymize_ip': true, 'cookie_expires': 29030400 })
+}
+
 function onConsentChanged() {
-  console.log(WcpConsent.siteConsent.getConsent());
+  if (!consentRequired()) {
```

Comment:
if consent isn't required, don't want to create the UA-62780441-30 gtag?

Reply:
fixed

Reply:
We'll need the same change in the LSP site PR.

Reply:
it already got merged before I saw this comment so I just updated directly

Reply:
Did I merge the LSP fix by accident?

Reply:
Oh, I see Dirk did. Thanks for fixing these. I'll kick off the WCP scans again to make sure we're green again.